### ISSUE TITLE
Cannot be installed from source on Mac

### ISSUE THREAD

**Issue by rinopo** [autarkper/Html5ValidatorPlus#6]
Hi,

I'm really interested in having this addon on my Firefox, but have been trying in vain.

It passes <code>ant build</code> with "BUILD SUCCESSFUL", but the resulting xpi is refused by Firefox with this message:

<blockquote>
HTML5 Validator Plus could not be installed because it is not compatible with Firefox 27.0.1.
</blockquote>

I happened to have an instance of Firefox 26, but the result was same as above.

Any help would be appreciated.

**Comment by autarkper** [comment]
Hi @rinopo and thank you for your interest in  HTML5 Validator Plus!

I'm unable to reproduce your problem on my Linux machine (Firefox 27.0.1). Does the browser console tell you anything that may point to the source of the problem?

**Comment by rinopo** [comment]
Thank you for the swift reply!
Unfortunately, the browser console remains blank.
Could you upload your working xpi to the [releases](https://github.com/autarkper/Html5ValidatorPlus/releases) page or elsewhere so that I can try?

**Comment by autarkper** [comment]
The releases page does not allow me to upload the xpi, but you can download the latest release from here: www.autark.se/Html5ValidatorPlus/html5validatorplus@autark.se-3.0.1.0.xpi.

**Comment by rinopo** [comment]
The one from your link worked just fine. Thank you so much!
Will check what was the difference between mine and yours.

**Comment by autarkper** [comment]
I'm happy the release xpi worked for you. Too bad the build on Mac does not work properly. It would be great if you could find out what is causing the problem.

**Comment by autarkper** [comment]
@rinopo: I have edited the issue description to point out that the problem is primarily Mac-related.

**Comment by rinopo** [comment]
It turned out to be an encoding problem.
I've learned that the character encoding in Java for OS X doesn't default to UTF-8 (in my case it was SJIS; it is said to be MacRoman elsewhere).
So I had to explicitly set the encoding to UTF-8 before running `ant`.

```
$ export ANT_OPTS=-Dfile.encoding=UTF8
$ ant build
```

Maybe it's an elementary thing, and sorry to have bothered you with that.
You've been very helpful. Thanks again!
